src/disco/css/InstallButton.scss

```diff
@@ -190,3 +170,19 @@ $installStripeColor2: #00C42E;
     }
   }
 }
+
+.progress {
+  background: transparentize(#3ef827, 0.50);
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 0;
+}
+
+@for $i from 1 through 100 {
+  [data-download-progress="#{$i}"] .progress {
+    width: #{percentage($i) / 100};
```

Comment:
Nice - yep some of that (the grey handle was from an old spec).

That should all be fixed. Definitely a good plan re: using transform + adding the transition.